packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -11,12 +11,12 @@ import * as PureRender from "pure-render-decorator";
 import * as React from "react";
 
 import * as Classes from "../../common/classes";
+import { Utils } from "../../common/utils";
 
-// amount in pixels that the content div width changes when truncated vs when
-// not truncated. Note: could be modified by styles
-// Note 2: this doesn't come from the width of the popover element, but the "right" style
-// on the div, which comes from styles
-const CONTENT_DIV_WIDTH_DELTA = 25;
+// Since we measure only the `textContent` of the cell to determine the
+// truncation state, we must account for the padding that is applied via CSS to
+// the cell.
+const CONTAINER_PADDING = 20;
```

Comment:
Why the change?

Reply:
ping @themadcreator

Reply:
Previously we had to consider the current truncation state when doing the calculation for content width because the element containing the text would have a different size if the "..." button was shown.

Now, since we're measuring with canvas, the size of the text content element is irrelevant. Instead we just need to consider the amount of padding between the parent element and the text content element.